misc-scripts/xref_mapping/XrefParser/EntrezGeneParser.pm

```diff
@@ -50,74 +50,29 @@ sub run {
   
 
   my $eg_io = $self->get_filehandle($file);
-  if ( !defined $eg_io ) {
-    print STDERR "ERROR: Could not open $file\n";
-    return 1;    # 1 is an error
-  }
+  croak "ERROR: Could not open $file\n" unless defined $eg_io;
```

Comment:
Errors are self-evidently errors when the croak fires. Is croak correct for this?